SampleApp/package.json

```diff
@@ -1,51 +1,51 @@
-{
-  "name": "ng6-sample-app",
-  "version": "1.0.0",
-  "scripts": {
-    "ng": "ng",
-    "start": "ng serve",
-    "build": "ng build",
-    "test": "ng test",
-    "lint": "ng lint",
-    "e2e": "ng e2e"
-  },
-  "private": true,
-  "dependencies": {
-    "@angular/animations": "^16.0.4",
-    "@angular/cdk": "^16.0.3",
-    "@angular/common": "^16.0.4",
-    "@angular/compiler": "^16.0.4",
-    "@angular/core": "^16.0.4",
-    "@angular/forms": "^16.0.4",
-    "@angular/material": "^16.0.3",
-    "@angular/platform-browser": "^16.0.4",
-    "@angular/platform-browser-dynamic": "^16.0.4",
-    "@angular/router": "^16.0.4",
-    "core-js": "^3.21.0",
-    "hammerjs": "^2.0.8",
-    "ng2-pdfjs-viewer": "file:ng2-pdfjs-viewer-16.0.4.tgz",
-    "rxjs": "^7.5.4",
-    "zone.js": "~0.13.0"
-  },
-  "devDependencies": {
-    "@angular-devkit/build-angular": "^16.0.4",
-    "@angular/cli": "^16.0.4",
-    "@angular/compiler-cli": "^16.0.4",
-    "@angular/language-service": "^16.0.4",
-    "@types/jasmine": "^3.10.3",
-    "@types/jasminewd2": "^2.0.10",
-    "@types/node": "^17.0.18",
-    "codelyzer": "^6.0.2",
-    "jasmine-core": "~4.0.0",
-    "jasmine-spec-reporter": "~7.0.0",
-    "karma": "~6.3.16",
-    "karma-chrome-launcher": "~3.1.0",
-    "karma-coverage-istanbul-reporter": "^3.0.3",
-    "karma-jasmine": "~4.0.1",
-    "karma-jasmine-html-reporter": "^1.7.0",
-    "protractor": "~7.0.0",
-    "ts-node": "^10.5.0",
-    "tslint": "~5.20.1",
-    "typescript": "^5.0.4"
-  }
-}
+{
+  "name": "ng6-sample-app",
+  "version": "1.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "^16.0.4",

```

Comment:
Might need to bump those version as well, it still as 16

Reply:
done